packages/ts-config-webpack-plugin/src/TsConfigWebpackPlugin.js

```diff
@@ -38,7 +38,7 @@ class TsConfigWebpackPlugin {
 		console.warn(
 			"Couldn't find a tsconfig.json in the current working directory.\nYou can either set the configFile path explicitly or create a new config:\n  npx tsc --init"
 		);
-		return path.resolve(__dirname, '../config/tsconfig.base.json');
+		return path.resolve(__dirname, path.join('..', 'config', 'tsconfig.base.json'));
```

Comment:
This is already done by path.resolve as you can see in the following example:

https://runkit.com/embed/c6p0czttkcpf

Reply:
It produces different results depending on the platform and there should not be any platform specific separators imo.

Reply:
I don't understand what you mean - `path.resolve` is always platform specific.

Reply:
I know but path.posix may result in different outputs than you expect.

Reply:
https://github.com/sindresorhus/file-url/commit/c77fefb96adee8d5a7e1a42cb2c257dafc10e9e2

I still see no issues with .replace (like I already did in other projects and others do too).

Reply:
Oh I see - but we don't use it here.

```
 path.resolve(__dirname, '../config/tsconfig.base.json')
```

And it works perfectly on windows:

<img width="547" alt="win7" src="https://user-images.githubusercontent.com/4113649/52912503-95c82480-32b2-11e9-8911-650d09e822a5.png">

Reply:
Please see the failing unit tests and the snapshots ;-)

Reply:
If this line here is not problematic I will revert it. But the others are.

Reply:
👍

Reply:
So far the original line does not product any failing tests on Windows. I have reverted it.